src/app/grid/grid-column-hiding-toolbar-style/grid-column-hiding-toolbar-style.component.scss

```diff
@@ -0,0 +1,80 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+/*BASIC THEMING*/
+
+$custom-column-hiding-theme: igx-column-hiding-theme(
+    $background-color: steelblue,
+    $title-color: gold
+);
+$custom-button: igx-button-theme($flat-text-color: gold, $disabled-color: black);
+
+/*INCLUSION by DEFAULT ViewEncapsulation strategy*/
```

Comment:
It would be better if the inclusion is added after all the styling approaches, so that only with the removing of the comments, the themes could be applied